app/femr/ui/views/admin/update/manage.scala.html

```diff
@@ -0,0 +1,99 @@
+@(currentUser: femr.common.dtos.CurrentUser,
+        viewModel: femr.ui.models.admin.configure.IndexViewModelGet,
+        assets: AssetsFinder)
+
+@import femr.ui.controllers.admin.routes.ConfigureController
+@import femr.ui.views.html.layouts.admin
+
+    @additionalStyles = {
+        <link rel="stylesheet" href="@assets.path("css/admin/configure.css")">
+    }
+
+@admin("Update Panel", currentUser, styles = additionalStyles, assets = assets) {
+    @helper.form(action = ConfigureController.managePost()) {
+
+        <div style="float:right">
```

Comment:
currently there is a gap with the columns, so maybe have it aligned in the upper right hand corner